requirements/base.in

```diff
@@ -1,3 +1,4 @@
+-c ../constraints.txt
```

Comment:
Could you move these new constraint lines between the file description comments and the requirements?  It's nice to have the summary of what the file is for right at the top, and matches what we've been doing elsewhere.

Reply:
Sure, done. And also re-ran `make upgrade` to verify I did not accidentally introduced a typo :-)